Apps/Encounter/test/unit/Services/EncounterServiceTests.cs

```diff
@@ -209,7 +210,7 @@ public async Task NoClaims()
                 GetIConfigurationRoot(),
                 MapperUtil.InitializeAutoMapper());
 
-            RequestResult<IEnumerable<EncounterModel>> actualResult = await service.GetEncounters(hdid);
+            RequestResult<IEnumerable<EncounterModel>> actualResult = await service.GetEncountersAsync(hdid);
```

Comment:
similar to above. See below as well, if this is intentional and the default will take hold you can ignore these comments.

Reply:
That is not required.  As it is handled by default.